File: CS/DXApplication5/MVVM/ViewModels/FindDialogViewModel.cs
using DevExpress.Mvvm;
using DevExpress.Mvvm.POCO;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace DXApplication5.MVVM.ViewModels {
    public class FindDialogViewModel {
        public FindDialogViewModel() { }
        public static FindDialogViewModel Create() {
            return ViewModelSource.Create<FindDialogViewModel>();
        }
        public virtual int SearchID { get; set; }
    }
}

File: VB/DXApplication5/MVVM/ViewModels/FindDialogViewModel.vb
Imports Microsoft.VisualBasic
Imports DevExpress.Mvvm
Imports DevExpress.Mvvm.POCO
Imports System
Imports System.Collections.Generic
Imports System.Linq
Imports System.Text
Imports System.Threading.Tasks

Namespace DXApplication5.MVVM.ViewModels
	Public Class FindDialogViewModel
		Public Shared Function Create() As FindDialogViewModel
			Return ViewModelSource.Create(Of FindDialogViewModel)()
		End Function
        Public Overridable Property SearchID As Int32
    End Class
End Namespace
